docs/source/release/templates/diffraction.rst

```diff
@@ -0,0 +1,41 @@
+===================
+Diffraction Changes
+===================
+
+.. contents:: Table of Contents
+   :local:
+
+Powder Diffraction
+------------------
+
+New features
+############
+.. amalgamate:: Diffraction/Powder/New_features
+
+Bugfixes
+############
+.. amalgamate:: Diffraction/Powder/Bugfixes
+
+
+Engineering Diffraction
+-----------------------
+
+New features
+############
+.. amalgamate:: Diffraction/Engineering/New_features
+
+Bugfixes
+############
+.. amalgamate:: Diffraction/Engineering/Bugfixes
+
+
+Single Crystal Diffraction
```

Comment:
In the Release Note Guide this section is a second Powder Diffraction. I guess this template is correct and the Release Note Guide wrong though

Reply:
The release notes are definitely incorrect! So glad you spotted this. I'll get it updated as part of this PR.